packages/arb-token-bridge-ui/src/hooks/useClaimWithdrawal.ts

```diff
@@ -33,18 +36,45 @@ export function useClaimWithdrawal(): UseClaimWithdrawalResult {
 
     setIsClaiming(true)
 
+    const childChainProvider = getProvider(tx.childChainId)
+    const txReceipt = await childChainProvider.getTransactionReceipt(tx.txId)
+    const l2TxReceipt = new L2TransactionReceipt(txReceipt)
+    const [event] = l2TxReceipt.getL2ToL1Events()
+
+    if (!event) {
+      setIsClaiming(false)
+      errorToast("Can't claim withdrawal: event not found.")
+      return
+    }
+
+    const extendedEvent: L2ToL1EventResultPlus = {
+      ...event,
+      sender: tx.sender,
+      destinationAddress: tx.destination,
+      l2TxHash: tx.txId,
+      type: tx.assetType,
+      value: utils.parseEther(tx.value ?? '0'),
+      tokenAddress: tx.tokenAddress || undefined,
+      outgoingMessageState: 1,
+      symbol: tx.asset,
+      decimals: 18,
```

Comment:
I don't think it matters that it's 18? It's a required property but it shouldn't be used to execute the claim.